### PR TITLE
Frequency Domain Analysis

### LINKED ISSUE #20655: Maintenance on Frequency Domain Analysis (FDA)
Some of the files in FDA need to be renamed to be lower case and the changes need to be followed through into the code. 

The code will also be cleaned up where possible and tests added if they were missed.

### PR DESCRIPTION
**Pull request by AnthonyLim23**
This was a clean up of the FDA GUI. It has included more comments and doc strings. Some refactoring to reduce repetition between files.  

**To test:**
Open Muon Analysis and load some data.
Just have a play around with the GUI

Fixes #20655. 

**Release Notes** 

There is no new functionality so does not need to be in the release notes. 
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [x] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Muon/fft_presenter.py [hunk 1]
```diff
@@ -100,6 +101,7 @@ def handleButton(self):
         self.thread.start()
         self.view.setPhaseBox()
 
+    # kills the thread at end of execution
     def handleFinished(self):
         self.activate()
         self.thread.deleteLater
```
Comment (soininen) [inline scripts/Muon/fft_presenter.py]: Is this really how the thread gets cleaned up? It is not actually a call?

File: scripts/Muon/fft_view.py [hunk 2]
```diff
@@ -127,7 +130,7 @@ def tableClick(self,row,col):
     def buttonClick(self):
         self.buttonSignal.emit()
 
-    #functions
+    # responses to cpmmands
```
Comment (soininen) [inline scripts/Muon/fft_view.py]: cpmmands -> commands

File: scripts/Muon/fft_model.py [hunk 3]
```diff
@@ -7,10 +7,16 @@
...
+    """
+    A class to wrap the different parts
+    of the FFT and its preprocessing.
+    This keeps the main FFT class simple.
+    """
     def __init__(self,FFT):
         self.name="FFT"
         self.model=FFT
 
+    #store the data in the wrapper for later
```
Comment (soininen) [inline scripts/Muon/fft_model.py]: This (and similarly below) could be a real docstring, I think.

**Review by soininen — CHANGES_REQUESTED** [review]
Played around with the Frequency domain analysis interface and as far as I can tell, it works fine. Except for the few minor comments the code looks good, too.

File: scripts/Muon/fft_presenter.py (hunk 1 above)
Comment (AnthonyLim23) [reply scripts/Muon/fft_presenter.py]: deleteLater is a slot, see https://stackoverflow.com/questions/28669195/qt-how-to-clean-up-a-qthread-when-you-do-not-subclass-it

Comment (soininen) [reply scripts/Muon/fft_presenter.py]: I know it's a slot, but I was merely wondering if you would actually need to call the attribute in Python, i.e. `self.thread.deleteLater()`. This is what I found people doing on the few examples I found on the Internet. Anyway, I'm not familiar with PyQt so I trust you know what you're doing here.

File: scripts/Muon/fft_model.py [hunk 4]
```diff
@@ -16,7 +16,9 @@ def __init__(self,FFT):
         self.name="FFT"
         self.model=FFT
 
-    #store the data in the wrapper for later
+    """
+    store the data in the wrapper for later
+    """
```
Comment (soininen) [inline scripts/Muon/fft_model.py]: I believe the docstrings should come after the function signature, e.g.
```python
    def loadData(self, inputs):
    """Store the data in the wrapper for later."""
```

**Review by soininen — APPROVED** [review]

**Pull request merged**